eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -0,0 +1,99 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [string] $TargetBranch,
+
+    [Parameter()]
+    [string] $SourceBranch,
+
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
```

Comment:
Is ```[ValidateNotNullOrEmpty()]``` helpful here since you set a default? I'd probably avoid setting a default for this particular script since ```./``` assumes you are invoking at the root of the source tree, but if you invoke from within ```eng/common/scripts``` it isn't going to work.

Reply:
At first I started down the path of coupling the script to its relative location in the repo directory structure (https://github.com/Azure/azure-sdk-tools/pull/1421/commits/b7ddb14f4f7c32880d83aadc17446fbdadce6ca1#diff-331922178311e6d75106a281f4a4e07841c3de3b514f77c5dbc766616ac4fc16L24). But it makes things more complex. The approach we're taking today gives us more flexibility. 

`ValidateNotNullOrEmpty` is only marginally helpful as it would require that some kind of path be fed into the `Test-Path` step a few lines into the script. I don't think its presence takes away from what we're doing here but if it complicates things we can take it out.